<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Mushfeques-MBP" failures="1" tests="5" name="Personal Loan" time="0.123" errors="0" timestamp="2022-03-05T11:47:31 EST">
  <testcase classname="test.Day1" name="Demo" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.Demo(Day1.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.Day2" name="pLoan" time="0.002"/>
  <testcase classname="test.Day4" name="apiLoginHomeLoan" time="0.008"/>
  <testcase classname="test.Day4" name="mobileLoginHomeLoan" time="0.01"/>
  <testcase classname="test.Day4" name="webLoginHomeLoan" time="0.007"/>
  <testcase classname="test.Day1" name="testCase2" time="0.0">
    <ignored/>
  </testcase> <!-- testCase2 -->
</testsuite> <!-- Personal Loan -->
